angular/BUILD.bazel

```diff
@@ -0,0 +1,29 @@
+load("@aspect_bazel_lib//lib:copy_to_bin.bzl", "copy_to_bin")
+load("@aspect_bazel_lib//lib:jq.bzl", "jq")
+load("@npm//:defs.bzl", "npm_link_all_packages")
+load(":defs.bzl", "ng_app")
+
+package(default_visibility = ["//visibility:public"])
+
+# Link npm packages
+npm_link_all_packages(name = "node_modules")
+
+# The Angular application
+ng_app(
+    name = "app",
+    project_name = "angular",
+    deps = ["//projects/lib-a"],
+)
+
+# Root config files used throughout
+copy_to_bin(
+    name = "angular",
+    srcs = ["angular.json"],
+)
+
+# NOTE: project dist directories are under the project dir unlike the Angular CLI default of the root dist folder
+jq(
+    name = "tsconfig",
+    srcs = ["tsconfig.json"],
+    filter = ".compilerOptions.paths |= map_values(map(gsub(\"^dist/(?<p>.+)$\"; \"projects/\"+.p+\"/dist\")))",
```

Comment:
could be followup- maybe we want this to go in a macro as well, so the .bzl file has the bazel magic, and this BUILD file reads a bit more like an end user would be okay maintaining